src/components/FeaturesDemo/WebBrowser/index.tsx

```diff
@@ -0,0 +1,88 @@
+import clsx from "clsx";
+import React, {
+    CSSProperties,
+    ForwardedRef,
+    forwardRef,
+    JSX,
+    ReactNode,
+    RefObject,
+    useImperativeHandle,
+    useRef,
+} from "react";
+
+import AngleIcon from "@site/static/icons/angle.svg";
+import PlusIcon from "@site/static/icons/plus.svg";
+import TabsIcon from "@site/static/icons/tabs.svg";
+import SpinnerIcon from "@site/static/icons/spinner.svg";
+import styles from "./styles.module.scss";
+
+export interface WebBrowserProps {
+    style?: CSSProperties;
+    className?: string;
+    page?: ReactNode;
+    isPageLoading?: boolean;
+    url?: string;
+}
+
+export interface WebBrowserApi {
+    setScrollTop: (offset: number) => void;
+    getScrollTop: () => number;
+    getUnsafePageRef: () => RefObject<HTMLDivElement>;
+}
+
+export const WebBrowser = forwardRef(function WebBrowserInternal(
+    props: WebBrowserProps,
+    ref: ForwardedRef<WebBrowserApi>,
+): JSX.Element {
+    const pageRef = useRef<null | HTMLDivElement>(null);
+
+    useImperativeHandle(ref, () => ({
+        setScrollTop: (offset: number): void => {
+            if (pageRef.current) {
+                pageRef.current.scrollTop = offset;
+            }
+        },
+        getScrollTop: (): number => {
+            return pageRef.current?.scrollTop ?? 0;
+        },
+        getUnsafePageRef: (): RefObject<HTMLDivElement> => {
+            return pageRef;
+        },
+    }));
+
+    return (
+        <div className={clsx("min-w-0", props.className)} style={props.style}>
+            <div className={clsx(styles.browser, "relative")}>
```

Comment:
why do you need `relative` here?

Reply:
So that page content could have absolutely positioned elements. For instance, we use it to have sticky header on app page or when using virtual cursor to click on buttons.